wasm/src/program/transaction.rs

```diff
@@ -0,0 +1,154 @@
+
+    #[test]
+    fn test_build_transaction() {
+        use std::include_bytes;
+        // let mint_buffer = include_bytes!("/Users/evanmarshall/.aleo/resources/mint.prover.6b3b2b0");
+        // ParameterProvider::store_bytes("MintProver".into(), mint_buffer.to_vec());
+        let transfer_buffer = include_bytes!("/Users/evanmarshall/.aleo/resources-backup/transfer.prover.837ad21");
```

Comment:
I changed: `resources` -> `resources-backup` to make sure we weren't accidentally using other files. When testing this locally, I renamed this directory to prevent any other file from relying on it.